### PR TITLE
add Dankort support

### LINKED ISSUE #100: Dankort is not recognized
### General information

* Library version: 
* Browser and OS 

### Issue description

I found this while using card-validator library. Dankort with IIN 5019 and 4571 is recognized as Maestro or Visa.

### PR DESCRIPTION
**Pull request by cassiocardoso**
Hi!

I was working on integrating this lib into another one that I'm maintaining and noticed that you had the Dankort issue flag tagged as `help wanted`, so here it is. 😄 

Since the changes are mostly in the card configuration and docs, I think this won't have a negative impact on the TS migration you've been working on.

Let me know if there's anything else needed.

### Proposal

Add `Dankort` support. (closes #100)

### Details

- Add Dankort card type configuration
- Update tests for detecting the new card type
- Update README with the relevant information for the `Dankort` type.

### CODE REVIEW

File: lib/card-types.js [hunk 1]
```diff
@@ -134,7 +134,8 @@ var cardTypes = {
     type: 'maestro',
     patterns: [
       493698,
-      [500000, 506698],
+      [5000, 5018],
+      [502000, 506698],
```
Comment (cassiocardoso) [inline lib/card-types.js:138]: I split the patterns here to make `5019` Dankort-exclusive. This didn't affect the other unit tests for Maestro.

**Comment by crookedneighbor** [comment]
We can't accept this without official documentation about the dankort ranges. Please provide that.

**Comment by cassiocardoso** [comment]
Hey @crookedneighbor I thought [this references](https://github.com/braintree/credit-card-type/issues/100#issuecomment-557500862) would be enough from your comments on the thread

**Comment by crookedneighbor** [comment]
LOL, you're right. Totally forgot the reference was in the original issue.

**Comment by cassiocardoso** [comment]
haha no worries :)

**Comment by cassiocardoso** [comment]
Hi @gesa, thanks for your review! Sounds good to me, let me know once the TS migration is merged and I can update this one :)

**Comment by crookedneighbor** [comment]
The typescript migration is complete.

**Comment by crookedneighbor** [comment]
Going to close this for now, but once you've merged in master, feel free to open a new PR.

**Review by shoaibmalik656 — APPROVED** [review]

**Pull request closed without merging**